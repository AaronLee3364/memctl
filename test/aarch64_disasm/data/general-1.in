# _pmap_find_phys, iPhone 7 10.1.1
FFFFFFF0071C5D28

                  A9BD57F6 A9014FF4
A9027BFD 910083FD AA0103F4 AA0003F3
F0001C75 F9442AA8 EB13011F
