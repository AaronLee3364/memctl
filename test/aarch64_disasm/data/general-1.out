fffffff0071c5d28  STP     X22, X21, [SP, #-48]!
fffffff0071c5d2c  STP     X20, X19, [SP, #16]
fffffff0071c5d30  STP     X29, X30, [SP, #32]
fffffff0071c5d34  ADD     X29, SP, #0x20
fffffff0071c5d38  MOV     X20, X1
fffffff0071c5d3c  MOV     X19, X0
fffffff0071c5d40  ADRP    X21, #0xfffffff007554000
fffffff0071c5d44  LDR     X8, [X21, #2128]
fffffff0071c5d48  CMP     X8, X19
